site/src/utils/formatDate.ts

```diff
@@ -0,0 +1,24 @@
+export const formatDate = (date: Date): string => {
+  if (isNaN(date.getTime())) {
+    return "To be seen";
```

Comment:
what was the intention with this ?
Maybe should read 'not yet defined' or something ?